main.go

```diff
@@ -160,12 +160,14 @@ func main() {
 		}
 	}
 	// Fetch the configuration for maximum number of API keys allowed per user.
-	maxAPIKeys, err := strconv.Atoi(os.Getenv(envMaxNumAPIKeysPerUser))
-	if err != nil {
-		log.Printf("Warning: Failed to parse %s env var. Error: %s", envMaxNumAPIKeysPerUser, err.Error())
-	}
-	if maxAPIKeys > 0 {
-		database.MaxNumAPIKeysPerUser = maxAPIKeys
+	if maxAPIKeysStr, ok := os.LookupEnv(envMaxNumAPIKeysPerUser); !ok {
```

Comment:
```suggestion
	if maxAPIKeysStr, ok := os.LookupEnv(envMaxNumAPIKeysPerUser); ok {
```
right?

Reply:
*facepalm*
yes...

Since I'm changing this, I'll use the opportunity to add a warning when the value is being ignored - Chris requested it for another parameter, it only makes sense to have that here as well.